src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -700,7 +700,26 @@ public class StripePaymentService : IPaymentService
                 DaysUntilDue = 1,
                 CollectionMethod = "send_invoice",
                 ProrationDate = prorationDate,
-            });
+            };
+
+            var customer = await new CustomerService().GetAsync(sub.CustomerId);
+            var taxRates = await _taxRateRepository.GetByLocationAsync(
+                new Bit.Core.Models.Table.TaxRate()
+                {
+                    Country = customer.Address.Country,
+                    PostalCode = customer.Address.PostalCode
+                }
+            );
+            var taxRate = taxRates.FirstOrDefault();
+            if (taxRate != null && !sub.DefaultTaxRates.Any(x => x.Equals(taxRate.Id)))
+            {
+                subUpdateOptions.DefaultTaxRates = new List<string>() 
+                { 
+                    taxRates.First().Id 
```

Comment:
don't you already have the `First()` tax rate stored in `taxRate` variable? Should avoid the extra LINQ extension here and re-use the ref you have already.